app/controller/DataController.php

```diff
@@ -210,6 +220,20 @@ function comment($request) {
             $comment->comment = $_POST['text'];
             $comment->user_id = $_SESSION['login'];
             $comment->save();
+            
```

Comment:
magst das ned in eine "service klasse" (stateless) auslagern ? dann ersparst die dir die redundanz und musst den code nur auf 1 stelle warten :D